src/scripts/components/PlayerControls.js

```diff
@@ -248,35 +247,6 @@ class PlayerControls extends Component<Props, State> {
     return currentVolume === 0
   }
 
-  setActivePlugin = (nextPlugin: PlayerPlugin): void => {
-    const { activePlugin, setActivePlugin } = this.props
-
-    setActivePlugin(
-      !nextPlugin || nextPlugin === activePlugin ? null : nextPlugin
-    )
-  }
-
-  renderPlugins () {
```

Comment:
`renderPlugins` now lives in `VideoOverlay.js`. This is because the plugins need to render within the bounds of the overlay, and the controls cannot easily access the bounds of the overlay.